Testscripts/Linux/ETHTOOL-GET-STATISTICS.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#############################################################################
+ChangeMTU(){
```

Comment:
()[space]{
and fix indentation inside function

Reply:
Fixed